cli/src/pcluster/resources/head_node/user_data.sh

```diff
@@ -61,6 +61,12 @@ function vendor_cookbook
   export HOME="${!HOME_BAK}"
 }
 [ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
+
+# deploy config files
+export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/aws/bin
```

Comment:
minor: why set an absolute path rather than appending on to the existing path?

Reply:
This is from legacy code, I think because we don't know what does existing path include and the path order in existing path may affect our execution, it stops at the first one it finds.